eclair-core/src/test/scala/fr/acinq/eclair/payment/MultiPartPaymentLifecycleSpec.scala

```diff
@@ -356,15 +358,15 @@ class MultiPartPaymentLifecycleSpec extends TestKitBaseClass with FixtureAnyFunS
     childPayFsm.send(payFsm, PaymentFailed(childId, paymentHash, Seq(RemoteFailure(route.amount, route.hops, Sphinx.DecryptedFailurePacket(b, TemporaryChannelFailure(channelUpdateBE1))))))
     // We update the routing hints accordingly before requesting a new route and ignore the channel.
     val routeRequest = router.expectMsgType[RouteRequest]
-    assert(routeRequest.assistedRoutes.head.head == routingHint)
+    assert(ChannelRelayParams.areSame(routeRequest.extraEdges.head.params, extraEdge.params, ignoreHtlcSize = true))
     assert(routeRequest.ignore.channels.map(_.shortChannelId) == Set(channelUpdateBE1.shortChannelId))
   }
 
   test("update routing hints") { () =>
-    val routingHints = Seq(
+    val extraEdges = Seq(
       Seq(ExtraHop(a, ShortChannelId(1), 10 msat, 0, CltvExpiryDelta(12)), ExtraHop(b, ShortChannelId(2), 0 msat, 100, CltvExpiryDelta(24))),
       Seq(ExtraHop(a, ShortChannelId(3), 1 msat, 10, CltvExpiryDelta(144)))
-    )
+    ).flatMap(Bolt11Invoice.toExtraEdges(_, c)).map(GraphEdge(_))
```

Comment:
You should change the test to create `BasicEdge`s instead of `ExtraHop`s and then map them to `GraphEdge`s.